controllers/machinehealthcheck_controller_test.go

```diff
@@ -187,10 +201,14 @@ var _ = Describe("MachineHealthCheck", func() {
 					},
 				},
 			}
-			mhc.Default()
+			// removed because to fix "when a Machine has no Node ref for less than the NodeStartupTimeout" issue
+			//mhc.Default()
```

Comment:
Could you elaborate a bit here? If we truly don't want this any more, I would prefer having the comment saying something like we would normally default the MHC here, but we can't because of xyz reasons.